tests/src/main/java/com/hazelcast/simulator/tests/map/GetAllMapTest.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (c) 2008-2013, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hazelcast.simulator.tests.map;
+
+import com.hazelcast.core.HazelcastInstance;
+
+import com.hazelcast.core.IMap;
+import com.hazelcast.logging.ILogger;
+import com.hazelcast.logging.Logger;
+import com.hazelcast.simulator.test.TestContext;
+import com.hazelcast.simulator.test.TestRunner;
+import com.hazelcast.simulator.test.annotations.*;
+import com.hazelcast.simulator.worker.selector.OperationSelectorBuilder;
+import com.hazelcast.simulator.worker.tasks.AbstractWorker;
+
+import java.util.Random;
+
+import static com.hazelcast.simulator.tests.helpers.HazelcastTestUtils.getOperationCountInformation;
+
+
+public class GetAllMapTest {
+
+    private static final ILogger log = Logger.getLogger(GetAllMapTest.class);
+    private static final OperationSelectorBuilder<Operation> operationSelectorBuilder = new OperationSelectorBuilder<Operation>();
+
+    private enum Operation {
+        PUT,
+        GET,
+    }
+
+    // properties
+    public int threadCount = 10;
+    public int keyCount = 1000;
+
+
+    private TestContext testContext;
+    private HazelcastInstance targetInstance;
+    private String basename = "tempVALE";
```

Comment:
This field is just useful if you want to run this test in parallel (not with multiple worker task but multiple times in the same testsuite). Then you can change the map name with this, but only if it is a public property. As private field it doesn't make sense.